README.md

```diff
@@ -181,15 +181,15 @@ You can also mix-in multiple data objects using object destructuring:
 
 **Structure:** `<input x-bind:[attribute]="[expression]">`
 
-`x-bind` sets the value of an attribute to the result of a JavaScript expression. The expression has access to all the keys of the component's data object, and will update every-time it's data is updated.
+`x-bind` sets the value of an attribute to the result of a JavaScript expression. The expression has access to all the keys of the component's data object, and will update every-time its data is updated.
 
 > Note: attribute bindings ONLY update when their dependencies update. The framework is smart enough to observe data changes and detect which bindings care about them.
 
 **`x-bind` for class attributes**
 
 `x-bind` behaves a little differently when binding to the `class` attribute.
 
-For classes, you pass in an object who's keys are class names, and values are boolean expressions to determine if those class names are applied or not.
+For classes, you pass in an object which keys are class names, and values are boolean expressions to determine if those class names are applied or not.
```

Comment:
```suggestion
For classes, you pass in an object whose keys are class names, and values are boolean expressions to determine if those class names are applied or not.
```

Reply:
You said that `I'm not sure if all of these changes are valid, grammatically.` But dude, I'm sure about.

Reply:
I didn't say that.

And what are you sure about?

Reply:
**I didn't say**: https://github.com/alpinejs/alpine/pull/58#issuecomment-572192579
**And what are you sure about**: I'm sure about the grammar"

In short, all the changes made in PR are not related to grammar, but a few, and among them, your pointing one. For this particular case, `that`, `which` and `who`, can be used but I used `which`.

Reply:
I did not make that comment. That is someone else.

Using `that`, `which` or `who` would all be incorrect. Each one of those words would make this an incorrect sentence. You're correct in suggesting `who's` be changed.  It should, however, be updated to `whose`.

All of your other grammar-related suggested changes are spot-on!

Reply:
Apologies